### PR TITLE
Mech Health Buff

### PR DESCRIPTION
**Pull request by Geevies**
* All mech components have had their health more than doubled.
* Cyborg components have had their health doubled.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/heavy_vehicle/components/_components.dm [hunk 1]
```diff
@@ -121,7 +121,7 @@
 		to_chat(user, "<span class='warning'>Turn \the [WT] on, first.</span>")
 		return
 	if(WT.remove_fuel(0, user))
-		var/repair_value = 15
+		var/repair_value = 30
```
Comment (Arrow768) [inline code/modules/heavy_vehicle/components/_components.dm:124]: Not in favor of the (undocumented) repair speed increase here.

If something is able to take more damage, it should also take longer to repair it.

Comment (Geevies) [inline code/modules/heavy_vehicle/components/_components.dm:124]:
```suggestion
		var/repair_value = 15
```

File: code/modules/heavy_vehicle/components/_components.dm [hunk 2]
```diff
@@ -145,7 +145,7 @@
 		if(QDELETED(CC) || QDELETED(src) || !CC.use(needed_amount))
 			return
 
-		repair_burn_damage(25)
+		repair_burn_damage(50)
```
Comment (Geevies) [inline code/modules/heavy_vehicle/components/_components.dm:148]:
```suggestion
		repair_burn_damage(25)
```

**Pull request merged**